ui/src/app/components/cromwell-configuration-panel.spec.tsx

```diff
@@ -109,6 +110,34 @@ describe('CromwellConfigurationPanel', () => {
     expect(onClose).toHaveBeenCalledTimes(1);
   });
 
+  it('should use the existing PD when creating', async () => {
+    const disk = stubDisk();
+    const wrapper = await component({
+      gkeAppsInWorkspace: [],
+      disk,
+    });
+    await waitOneTickAndUpdate(wrapper);
+
+    const spyCreateApp = jest
+      .spyOn(appsApi(), 'createApp')
+      .mockImplementation((): Promise<any> => Promise.resolve());
+    const startButton = wrapper
+      .find('#Cromwell-cloud-environment-create-button')
+      .first();
+
+    startButton.simulate('click');
+    await waitOneTickAndUpdate(wrapper);
+    expect(spyCreateApp).toHaveBeenCalledTimes(1);
+    expect(spyCreateApp).toHaveBeenCalledWith(
+      WorkspaceStubVariables.DEFAULT_WORKSPACE_NS,
+      {
+        ...defaultCromwellConfig,
+        persistentDiskRequest: disk,
+      }
+    );
```

Comment:
Since you've already asserted that this is called only once, I'd consider eschewing the workspace name and most of the config and asserting the disk only. The purpose behind that is to make it as clear as possible to the reader that what I care about here is that the disk is included in the request.

Reply:
Great idea, done. I've definitely been overusing `toHaveBeenCalledWith`.